docs/detections/prebuilt-rules-management.asciidoc

```diff
@@ -108,18 +107,85 @@ NOTE: The *Rule Updates* tab doesn't appear if all your installed prebuilt rules
 [role="screenshot"]
 image::images/prebuilt-rules-update.png[The Rule Updates tab on the Rules page]
```

Comment:
image to be updated after [issue](https://github.com/elastic/kibana/issues/210358) is merged.

Reply:
Addressed in [b5b5fc4](https://github.com/elastic/security-docs/pull/6568/commits/b5b5fc41d6ddeeaad0dfa58ea62a85d6411920cd).